robot/EDA/resources/TriggerHandlersPageObject.py

```diff
@@ -0,0 +1,195 @@
+import time
+
+from selenium.webdriver.common.keys import Keys
+from cumulusci.robotframework.pageobjects import HomePage
+from cumulusci.robotframework.pageobjects import BasePage
+from cumulusci.robotframework.pageobjects import DetailPage
+
+from cumulusci.robotframework.pageobjects import pageobject
+from locators import trigger_handlers_locators
+
+
+
+@pageobject("Home", "TriggerHandlers")
+class TriggerHandlersHomePage(BasePage):
+    object_name = None
+
+    @property
+    def eda(self):
+        return self.builtin.get_library_instance('EDA')
+
+    def _is_current_page(self):
+        """ Verify we are on the Trigger Handlers page
+            by verifying that the header title is 'Trigger Handlers'
+        """
+        self.selenium.page_should_contain_element(
+            trigger_handlers_locators["header"],
+            message="Header with text 'Trigger Handlers' is not available on the page"
+        )
+
+    def check_if_element_exists(self, xpath):
```

Comment:
Is this intended to be a keyword? The only place I see it being used is from within other keywords. If it's just a helper function and not a keyword that you expect to use in a test, I recommend naming it with a leading underscore (eg: `_check_if_element_exists` so that it's clear that it's not intended to be used in a test.

Reply:
removed - not being used.

Reply:
I removed this file as it's no longer being used, nor was being updated.

Reply:
I removed all references to this file across all of verify_preferredphone